### DISCUSSION TITLE
Can't get bookmarklet to work

### DISCUSSION THREAD

**Discussion by ptizzl** [mealie-recipes/mealie#738] [Q&A]
Hello,

I followed instructions by copying the code from the document site here: https://hay-kot.github.io/mealie/documentation/recipes/recipes/ into the bookmarklet generator and changed the URL. I've tried both my subdomain (mealie.domain.cloud) as well as my internal ip address (192.168.x.x:9925) but all that happens when I click the bookmarklet is that it opens mealie in a new window. 

Any suggestions?

**Comment by zierbeek** [comment]
Hi! If you update to the dev branch, you will be able to share it directly from the upper menu.

**Reply by ptizzl** [reply]
Hello, thanks for the reply. However, I'm not trying to share the recipe, rather I'm trying to find a fast way to import them. My idea was that if I could use the bookmarklet, it would be easier for my wife and I to each add recipes from our phones.

**Reply by zierbeek** [reply]
hi ! that is exactly what I meant, an easy way to implement it in mealie. It is named the web share api. You should just install the mealie progressive web app on your phone :)

**Reply by zierbeek** [reply]
and run the dev tag ofc

**Reply by ptizzl** [reply]
That sounds great. Sadly, I'm very novice with this stuff so I have a few questions. 
1) how would I get the dev branch? I installed this through a docker-compose file. 
2) how do I install the progressive web app on an iphone? What even is the progressive web app? 
3) what's dev tag mean?

**Reply by ptizzl** [reply]
Just to try and not be completely helpless, I duplicated everything and made a brand new container with image: hkotel/mealie:dev to try and see if that would work, and it seems to have the same exact version 0.52 showing, and application mode shows production, so I obviously don't know how to pull the dev version :)

**Reply by zierbeek** [reply]
Aha yeah got ya! The 0.5.2 keeps to be shown, if you have put the :dev tag, all should work fine!  The dev tag is the most recent version of mealie which is before release, sometimes it brakes ;) 

a progressive web is a webpage installed as an app, it is not a real shortcut but it feels more like an app. On android, just go to your mealie instance and it should prompt to install at the bottom.

If you are on iOS, then you should create a shortcut.
